bears/markdown/MarkdownBear.py

```diff
@@ -39,7 +44,8 @@ def create_arguments(filename, file, config_file,
                          markdown_list_increment: bool=True,
                          markdown_horizontal_rule: str='*',
                          markdown_horizontal_rule_spaces: bool=False,
-                         markdown_horizontal_rule_repeat: int=3):
+                         markdown_horizontal_rule_repeat: int=3,
+                         markdown_maximum_line_length: int=80):
```

Comment:
this should be `max_line_length`, which is a commonly used setting by many bears.